wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -5,46 +5,84 @@ sidebar_position: 2
 
 # Run a development network
 
-You can run a personal Ethereum development network using [Ganache](https://www.trufflesuite.com/ganache),
+You can run a personal Ethereum development network using [Hardhat](https://hardhat.org/),
 which allows you to develop a dapp in a secure test environment.
 
 :::note
-When using a local development blockchain such as Ganache or
+When using a local development blockchain such as Hardhat or
 [anvil](https://book.getfoundry.sh/anvil/#overview-of-anvil), your node must calculate gas to make
 transactions on MetaMask.
 :::
 
-## Connect to Ganache
-
-Follow the [Ganache quickstart](https://trufflesuite.com/docs/ganache/quickstart/) to set
-up a development network.
-
-When you create a Ganache workspace, enter your MetaMask seed phrase into
-the **Account & Keys** setting.
-Ganache automatically gives each of your first 10 accounts 100 test ether (you can configure
-these numbers in **Accounts & Keys**), which makes it easy to start development.
-
-:::caution important
-Your seed phrase controls all your accounts, so we recommend keeping at least one seed phrase for
-development, separate from any used to store real value.
-You can manage multiple seed phrases by using multiple browser profiles, each with its own
-MetaMask installation.
-:::
-
-In the **Server** setting of your workspace, find the hostname and port of your Ganache
-network, which comprises the RPC URL of your network:
-
-```text
-http://<hostname>:<port>
-```
-
-In the MetaMask extension, connect to your Ganache network:
-
-1. Select the network you're currently connected to.
-1. Select **Add network**.
-1. Select **Add a network manually**.
-1. Enter the RPC URL of your network.
-1. Enter MetaMask's default [chain ID](../connect/detect-network.md#chain-ids) for Ganache, `1337`.
+## Connect to Hardhat
+
+Follow these steps to connect MetaMask to a Hardhat development network.
+
+1. Follow the [Hardhat Quick Start](https://hardhat.org/hardhat-runner/docs/getting-started#quick-start)
+    to set up a Hardhat development network.
+
+2. Create a new
+    [MetaMask seed phrase](https://support.metamask.io/hc/en-us/articles/360060826432-What-is-a-Secret-Recovery-Phrase-and-how-to-keep-your-crypto-wallet-secure#:~:text=Your%20Secret%20Recovery%20Phrase%20(SRP,are%20connected%20to%20that%20phrase.))
+    specifically for development.
+
+    :::caution important
+    Your seed phrase controls all your accounts, so we recommend keeping at least one seed phrase for
+    development, separate from any used to store real value.
+    You can manage multiple seed phrases by using multiple browser profiles, each with its own
+    MetaMask installation.
+    :::
+
+3. In your `hardhat.config.js` file, in the `networks.hardhat` configuration:
```

Comment:
When testing Hardhat I noticed the `hardhat.config.js` doesn't have any networks defined by default. So should this be, "In your `hardhat.config.js` file, create a `networks` section..."